easybuild/easyconfigs/p/pymatgen/pymatgen-2022.9.21-foss-2022a.eb

```
@@ -0,0 +1,70 @@
+easyblock = 'PythonBundle'
+
+name = 'pymatgen'
+version = '2022.9.21'
+
+homepage = 'https://pypi.python.org/pypi/pymatgen'
+description = """Python Materials Genomics is a robust materials analysis code that defines core object
+ representations for structures and molecules with support for many electronic structure codes."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('PyYAML', '6.0'),
+    ('matplotlib', '3.5.2'),
+    ('networkx', '2.8.4'),
+    ('sympy', '1.11.1'),
+    ('plotly.py', '5.10.0'),
```

```suggestion
    ('plotly.py', '5.12.0'),
```

Use 5.12.0, from  #17146